apps/meteor/client/views/root/hooks/useWebRTC.ts

```diff
@@ -0,0 +1,37 @@
+import { useStream, useUserId } from '@rocket.chat/ui-contexts';
+import { useEffect } from 'react';
+
+import type { CandidateData, DescriptionData, JoinData } from '../../../../app/webrtc/client/WebRTCClass';
+import { WebRTC } from '../../../../app/webrtc/client/WebRTCClass';
+
+export const useWebRTC = () => {
+	const uid = useUserId();
+	const notifyUser = useStream('notify-user');
+
+	useEffect(() => {
+		if (!uid) return;
+
+		const handleNotifyUser = (type: 'candidate' | 'description' | 'join', data: CandidateData | DescriptionData | JoinData) => {
+			if (data.room == null) return;
+
+			const webrtc = WebRTC.getInstanceByRoomId(data.room);
+
+			if (!webrtc) return;
+
+			const handleWebRTCTypeObject = {
+				candidate: () => webrtc.onUserStream('candidate', data as CandidateData),
+				description: () => webrtc.onUserStream('description', data as DescriptionData),
+				join: () => webrtc.onUserStream('join', data as JoinData),
+				default: () => console.warn(`WebRTC: Received unexpected event type: ${type}`),
+			};
+
+			(handleWebRTCTypeObject[type] || handleWebRTCTypeObject.default)();
+		};
+
+		const handleStopNotifyUser = notifyUser(`${uid}/webrtc`, handleNotifyUser);
+
+		return () => {
+			handleStopNotifyUser();
+		};
```

Comment:
```suggestion
		return notifyUser(`${uid}/webrtc`, handleNotifyUser);
```

Reply:
Also any reason that you stopped using the constant `WEB_RTC_EVENTS.WEB_RTC` here? I think WEBRTC will probably/is deprecated, but if we ever change the event name we just have tu update the constants.

Reply:
Just a small oversight, was having some problems with typing and at some point removed the constant to see if it fixed the issue but forgot to add it back when I found the actual culprit.